lisa/nic.py

```diff
@@ -143,7 +145,7 @@ def get_unpaired_devices(self) -> List[str]:
         return [x.upper for x in self.nics.values() if not x.lower]
 
     def get_upper_nics(self) -> List[str]:
-        return [x.upper for x in self.nics.values() if x.lower]
+        return [x.upper for x in self.nics.values()]
```

Comment:
Please check with Matthew, if it's a breaking change.

Reply:
no this shouldn't be breaking change. we need to remove this coupling if we wnat to use this class as generic class for both Synthetic and Sriov nic.